src/offline-support/types.ts

```diff
@@ -399,14 +399,17 @@ export type PendingTaskTypes = {
   deleteReaction: 'delete-reaction';
   sendReaction: 'send-reaction';
   sendMessage: 'send-message';
+  createDraft: 'create-draft';
+  deleteDraft: 'delete-draft';
 };
 
 // TODO: Please rethink the definition of PendingTasks as it seems awkward
 export type PendingTask = {
   channelId?: string;
   channelType?: string;
-  messageId: string;
+  messageId?: string;
   id?: number;
+  parentId?: string;
```

Comment:
I am wondering whether in the future we could be working with parent of something else than a message and then `parentId` would not have enough specificity. I am thinking whether `threadId` would be less ambivalent.

Reply:
Sure, this can make sense. I'll do this.

Reply:
Resolved [here](c52f328974b107ee9bf15a69d835b435a4e1c83c).